packages/components/src/data-model/word-query/lexical-data/data-objects/tufts-morphology-data.js

```diff
@@ -0,0 +1,60 @@
+import LexicalDataResult from '@comp/data-model/word-query/lexical-data/result/lexical-data-result.js'
+import { LanguageModelFactory as LMF, Logger, HomonymGroup } from 'alpheios-data-models'
+import { ClientAdapters } from 'alpheios-client-adapters'
+import WordQueryErrorCodes from '@comp/data-model/word-query/error/word-query-error-codes.js'
+import ErrorMapper from '@comp/data-model/word-query/error/error-mapper.js'
+
+export default class TuftsMorphologyData {
+  constructor ({ word, language, clientId, clearShortDefs } = {}) {
+    this._word = word
+    this._language = language
+    this._clientId = clientId
+    this._clearShortDefs = clearShortDefs
+  }
+
+  async retrieve () {
```

Comment:
since the retrieve call is always passed a lexical data object, maybe it should be here even though it's not used (and will be empty)?

Reply:
Will add that